### DISCUSSION TITLE
ImportError: Could not import PIL.Image

### DISCUSSION THREAD

**Discussion by Githubmuls** [teamalgoritma/community#321] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/107347991/191539561-7c892b5c-28b0-4ee8-967d-23fda24f8603.png)

Ada keterangan requires PIL padahal pada env yang saya gunakan sudah memiliki Pillow

![image](https://user-images.githubusercontent.com/107347991/191540192-ee4afc02-ddcd-4cc7-b4c2-a62be1ae461b.png)

**Comment by NabiilahArdini** [comment, marked as answer]
Halo @Githubmuls,
Pertanyaan Bapak sepertinya menyerupai diskusi ini: https://stackoverflow.com/questions/59169441/importerror-could-not-import-pil-image/ Apakah bisa dicoba untuk melakukan install pillow via conda. Caranya dengan run code berikut di terminal/anaconda prompt pada environment Bapak: `conda install pillow`
